test/signUpEndpoint.test.js

```diff
@@ -0,0 +1,157 @@
+const request = require('supertest');
+const Chance = require('chance');
```

Comment:
check if you can use this:

`const chance = require('chance')();` to avoid the constructor after.